src/store/store.ts

```diff
@@ -23,17 +23,18 @@ export const storeConfig = configureStore({
 })
 
 export interface Card {
-  columnID: string
   text: string
 }
 
 export interface Column {
   title: string
+  cardsIds: string[]
 }
 
 export interface DBSchema {
   cards: Record<string, Card> | undefined
   columns: Record<string, Column> | undefined
+  columnsOrder: string[] | undefined
```

Comment:
You have it inside `userData`, so you can remove that after adding suggestion from earlier comment regarding `store`.